lib/modules/platform/gerrit/index.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { stripIndent } from 'common-tags';
```

Comment:
```suggestion
import { codeBlock } from 'common-tags';
```

we use `codeBlock` in all other places